### ISSUE TITLE
Typescript compile error in tl\custom\forward.d.ts

### ISSUE THREAD

**Issue by aguerrieri82** [gram-js/gramjs#160]
I have the following typescript error in `tl\custom\forward.d.ts` (and other 4 files)

`ts-mixer/dist/types/index"' has no exported member 'Class'`

### Error line:

```typescript
import("ts-mixer/dist/types").Class
```
 
###  Version

telegram npm:  **1.9.0**
ts-mixer module resolved version: **5.4.1**
typescript compiler: **4.4**
build systems: **MSBuild**
 
### package.json:
 
```json
 "devDependencies": {
    "barrelsby": "2.2.0",
    "source-map": "0.7.3",
    "source-map-loader": "3.0.0",
    "terser-webpack-plugin": "5.2.4",
    "tslint": "6.1.3",
    "tslint-etc": "1.13.10",
    "typescript": "4.4.3",
    "webpack": "5.52.1",
    "webpack-cli": "4.8.0"
  },
  "dependencies": {
    "@microsoft/signalr": "5.0.9",
    "telegram": "1.9.0"
  }
```
### tsconfig.json

```json
  {
  "compilerOptions": {
    "noImplicitAny": false,
    "noEmitOnError": false,
    "sourceMap": true,
    "target": "ES2017",
    "module": "ES6",
    "moduleResolution": "Node",
    "outDir": "obj/js",
    "allowSyntheticDefaultImports": true,

  },
  "exclude": [
    "node_modules"
  ]
```

**Comment by painor** [comment]
can you try using something similar to https://github.com/gram-js/gramjs/blob/master/tsconfig.json ?

**Comment by aguerrieri82** [comment]
solved by forcing ts-mixer version to 5.4.0 in my package.json
version  5.4.1 has  breaking change in dist folder;
